test/tests/functional/pbs_multi_sched.py

```diff
@@ -825,8 +825,13 @@ def test_fairshare_usage(self):
         sc1_fs_user4 = self.scheds['sc1'].query_fairshare(name=str(TEST_USER4))
         self.assertEqual(sc1_fs_user4.usage, 1)
         # Restart the scheduler
+        t = time.time()
         self.scheds['sc1'].restart()
         # Check the multisched 'sc1' usage file whether it's updating or not
+        self.assertTrue(self.scheds['sc1'].isUp())
+        self.server.manager(MGR_CMD_SET, SCHED, {'scheduling': 'True'},
+                            id='sc1')
+        self.scheds['sc1'].log_match("Scheduler is reconfiguring", starttime=t)
```

Comment:
With your above change, this won't happen any more.  We no longer send a configure if we only set scheduling.

Reply:
I was running into an issue where after the restart, the scheduler would schedule jobs, then the on next scheduling cycle it would reconfigure. I assume this is due to the scheduler coming back up from being down.

Reply:
I don't know why it would reconfigure.  When it comes up it just queries the sched attributes directly.  It doesn't do a configure.  The server log might help you there, or PTL is HUPing the scheduler for some reason.